snapshot/git/gitdb/setup.go

```diff
@@ -38,6 +38,9 @@ func (m module) Install(b *ice.MagicBag) {
 		func() AutoUploadDest {
 			return AutoUploadBundlestore
 		},
+		func() string {
```

Comment:
honestly don't really understand why we need this... is this dbTempDir?

Reply:
oss apiserver main.go implicitly calls MakeDBNewRepo here via ice, and that func needs a string for (a preexisting) tmpdir path. In closed source we call it explicitly, without ice

Reply:
Does that mean that the oss MakeDBNewRepo will use / as temp then? Should we do os.TempDir() here instead of ""?

Reply:
It will effectively pass "" through which will result in ioutil.TempDir("", ...) calls, which is equivalent to using os.TempDir()

Reply:
ex:

> $ go run github.com/twitter/scoot/apiserver
INFO[0000] Making new FileStore at dir: /var/folders/y0/6frrxtkn495dgbzr_s_7zcgr0000gp/T/bundles074177289  sourceLine="store/file_store.go:26"
INFO[0000] Starting new bundlestore.Server with root: /var/folders/y0/6frrxtkn495dgbzr_s_7zcgr0000gp/T/bundles074177289  sourceLine="bundlestore/server.go:34"
INFO[0000] git.NewRepository: /var/folders/y0/6frrxtkn495dgbzr_s_7zcgr0000gp/T/gitdb-repo-541018836, top: /private/var/folders/y0/6frrxtkn495dgbzr_s_7zcgr0000gp/T/gitdb-repo-541018836  sourceLine="repo/repo.go:132"